### DISCUSSION TITLE
Lazysar: Visualization of sysstat data in terminal

### DISCUSSION THREAD

**Discussion by brablc** [sysstat/sysstat#394] [Show and tell]
I use sar/sysstat for more than a decade but always missed an easy way how to visualize data in terminal. Therefore I created a small tool https://github.com/brablc/lazysar . It uses https://github.com/zellij-org/zellij for multipanel support and https://github.com/tammoippen/plotille for charting and it can be used to display various charts from multiple hosts and use prompt to zoom to specific date/time for all charts at once. It even supports real-time charts.

Thanks for sysstat!

**Comment by sysstat** [comment]
Seems cool ! I shall give it a try...
